### ISSUE TITLE
course landing page images

### ISSUE THREAD

**Issue by tfg250** [IllinoisRef/mechref#38]
The not-simple view of each course landing page shows what looks like placeholder images for each topic, how can I help get these replaced with a more relevant image?

**Comment by tfg250** [comment]
My concern is when we "unveil" this page next Monday. Any image would be better than the placeholders. Or maybe we just disable that view for now until the images are in place?

**Comment by mekersh** [comment]
I like them - makes the page visually interested.  But I agree with @tfg250 that we should replace them.  I'll offer this as a temporary solution - we can make them more customized to the specific sub pages later.  I'm also going to make a version of this for the "home" page so that the connection is clearer.  
![solids](https://github.com/user-attachments/assets/99dd8c18-64a4-4465-b9e2-8ea6de5d5c59)
![dynamics](https://github.com/user-attachments/assets/f887271e-6486-495a-9f46-b8fde003c13f)
![statics](https://github.com/user-attachments/assets/15f28ed9-d393-4e6b-9af4-7c029e496af9)

**Comment by mekersh** [comment]
Option for the "home" page 
![home](https://github.com/user-attachments/assets/fa707ee2-e09e-4878-b44e-0cd8eed370d3)

**Comment by mekersh** [comment]
@mwest1066 @tfg250 are you ok with these as placeholders for now?

**Comment by tfg250** [comment]
yeah, looks good. @jcrayb is there a way to change the default view to be the simplified view until the smaller images are replaced?

**Comment by jcrayb** [comment]
Yes, should be fairly easy.

**Comment by jcrayb** [comment]
Ok, I've merged changes that make the simplified view the default and prevents user from going back to the view with the images.
